lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8441,6 +8449,26 @@ ClangImporter::Implementation::loadAllMembers(Decl *D, uint64_t extra) {
     loadAllMembersOfObjcContainer(D, objcContainer);
     return;
   }
+
+  auto namespaceDecl =
+      dyn_cast_or_null<clang::NamespaceDecl>(D->getClangDecl());
```

Comment:
I think we can assume it has a Clang decl if it got here, so just `dyn_cast` is fine.

Reply:
Actually, this falls into loadAllMembersIntoExtension which is like so:
ExtensionDecl of NominalDecl that contains the ClangDecl.  There are places in this file that construct ExtensionDecls without clang decls.

Reply:
Oh, right, okay.